README.md

```diff
@@ -79,21 +81,21 @@ await ciscoTracing.init(userOptions);
 
 #### Configure custom trace exporter
 
-> Cisco OpenTelemetry Distribution supports configure multiple custom exporters.
+> Cisco OpenTelemetry Distribution supports configure multiple custom exporters. In that case, ciscoToken is ommited, and instead a custom header is given as described below:
```

Comment:
I suggest:
Cisco OpenTelemetry Distribution supports configuring multiple custom exporters. Note that you will need to handle authorization.